restful_resource_bugsnag.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "bugsnag", "~> 5"
   spec.add_dependency "restful_resource", "~> 2"
 
-  spec.add_development_dependency "bundler", "~> 1.12"
+  spec.add_development_dependency "bundler", "~> 2.0"
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "rspec", "~> 3.0"
   spec.add_development_dependency "webmock", "~> 2.0", ">= 2.0.3"
```

Comment:
I would remove all of the version requirements, with the lock file this isn't so important.